### ISSUE TITLE
Cannot pull one image

### ISSUE THREAD

**Issue by Yajo** [docker/compose#1726]
In my docker-compose.yml file there are some images, but one of them cannot be pulled. The other ones present no problem:

```
$ ./docker-compose pull
Pulling wdb (docker.io/yajo/wdb-server:latest)...
Traceback (most recent call last):
  File "./docker-compose", line 9, in <module>
    load_entry_point('docker-compose==1.3.3', 'console_scripts', 'docker-compose')()
  File "/home/jllopis/Documentos/devel/docker/odoo-grupoesoc/pyenv/lib/python2.7/site-packages/compose/cli/main.py", line 32, in main
    command.sys_dispatch()
  File "/home/jllopis/Documentos/devel/docker/odoo-grupoesoc/pyenv/lib/python2.7/site-packages/compose/cli/docopt_command.py", line 21, in sys_dispatch
    self.dispatch(sys.argv[1:], None)
  File "/home/jllopis/Documentos/devel/docker/odoo-grupoesoc/pyenv/lib/python2.7/site-packages/compose/cli/command.py", line 34, in dispatch
    super(Command, self).dispatch(*args, **kwargs)
  File "/home/jllopis/Documentos/devel/docker/odoo-grupoesoc/pyenv/lib/python2.7/site-packages/compose/cli/docopt_command.py", line 24, in dispatch
    self.perform_command(*self.parse(argv, global_options))
  File "/home/jllopis/Documentos/devel/docker/odoo-grupoesoc/pyenv/lib/python2.7/site-packages/compose/cli/command.py", line 66, in perform_command
    handler(project, command_options)
  File "/home/jllopis/Documentos/devel/docker/odoo-grupoesoc/pyenv/lib/python2.7/site-packages/compose/cli/main.py", line 235, in pull
    insecure_registry=insecure_registry
  File "/home/jllopis/Documentos/devel/docker/odoo-grupoesoc/pyenv/lib/python2.7/site-packages/compose/project.py", line 285, in pull
    service.pull(insecure_registry=insecure_registry)
  File "/home/jllopis/Documentos/devel/docker/odoo-grupoesoc/pyenv/lib/python2.7/site-packages/compose/service.py", line 713, in pull
    insecure_registry=insecure_registry)
  File "/home/jllopis/Documentos/devel/docker/odoo-grupoesoc/pyenv/lib/python2.7/site-packages/docker/client.py", line 590, in pull
    repository, insecure=insecure_registry
  File "/home/jllopis/Documentos/devel/docker/odoo-grupoesoc/pyenv/lib/python2.7/site-packages/docker/auth/auth.py", line 60, in resolve_repository_name
    return expand_registry_url(parts[0], insecure), parts[1]
  File "/home/jllopis/Documentos/devel/docker/odoo-grupoesoc/pyenv/lib/python2.7/site-packages/docker/auth/auth.py", line 39, in expand_registry_url
    "HTTPS endpoint unresponsive and insecure mode isn't enabled."
docker.errors.DockerException: HTTPS endpoint unresponsive and insecure mode isn't enabled.
$ ./docker-compose --version
docker-compose version: 1.3.3
CPython version: 2.7.10
OpenSSL version: OpenSSL 1.0.1k-fips 8 Jan 2015
```

But Docker itself pulls it without problem:

```
docker pull docker.io/yajo/wdb-server:latest
latest: Pulling from docker.io/yajo/wdb-server
[...]
Status: Downloaded newer image for docker.io/yajo/wdb-server:latest
```

**Comment by aanand** [comment]
Does it work if you pass `--allow-insecure-ssl`?

**Comment by Yajo** [comment]
Yes.

**Comment by aanand** [comment]
OK. Once https://github.com/docker/docker-py/pull/681 is merged, we'll be able to deprecate that flag, but for now it's necessary.

**Comment by aanand** [comment]
Fixed by #1736 and released in [Compose 1.4.0 RC2](https://github.com/docker/compose/releases/tag/1.4.0rc2).
